test_car.py

```diff
@@ -216,9 +216,9 @@ def test_forward_then_reverse_acceleration_basic(car_object_basic):
     assert car_object_basic.check_velocity() == 0
     assert round(car_object_basic.check_distance(), 2) == 58.8
     car_object_basic.accelerate_forward(-5,3)
-    car_object_basic.accelerate_reverse(3, 1)
+    car_object_basic.accelerate_reverse(3, 1) 
     assert car_object_basic.check_velocity() == 12
-    assert round(car_object_basic.check_distance(), 3) == 139.8
+    assert round(car_object_basic.check_distance(), 3) == 97.8
 
```

Comment:
Here I got 94.8 instead of 97.8

When velocity is 0 and distance is 58.8 we want to accelerate forward 5 distance units for 3 time units.
New Velocity = 5 * 3 + 0 = 15
Distance = 0.5 * 5 * 3**2 + 0 * 3 + 58.8 = 81.3

Then we accelerate in the reverse direction for 3 distance units and 1 time unit. 
New Velocity = -3 * 1 + 15 = 12
New Distance = abs(0.5 * -3 * 1**2 + 15 * 1) + 81.3 = 94.8